python_modules/libraries/dagster-dbt/dagster_dbt/core/resources_v2.py

```diff
@@ -956,6 +1040,24 @@ def fetch_row_counts(
         """
         return self.attach_metadata(_fetch_row_count_metadata)
 
+    @public
+    @experimental
+    def fetch_column_metadata(
```

Comment:
Let's be more explicit here?

```suggestion
    def fetch_column_lineage(
```

And maybe for folks who don't want column lineage, but want materialization level column schema, there could be a separate `def fetch_column_schema` method?

Or there could just be a `fetch_column_schema` method

```suggestion
    def fetch_column_schema(with_column_lineage: bool = True)
```

Reply:
Yeah, I think the latter makes sense, since the metadata is a prerequisite to lineage so it's a little tricky to split them (even though it's conceptually a bit cleaner)